deploy/k3d/k3d.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: k3d.io/v1alpha2
+kind: Simple
+servers: 1
+network: k3d
+
+kubeAPI:
+  hostIP: "0.0.0.0"
+  hostPort: "6443"
+
+ports:
+  - port: 30080:30080
```

Comment:
do we need both ports?

Reply:
We mostly need the ngsa nodePort to check healthz or version endpoint for brust header.
But sometimes when developing burstService, we also need the burst service nodePort.

Reply:
Port forwarded in devcontainer.json and properly labeled them.